### PR TITLE
Milestone 5 - Math magicians: Full website

### PR DESCRIPTION
**Pull request by grauJavier**
## 🌟 Completed Project Requirements
- A website comprising three pages, namely `Home`, `Calculator`, and `Quote`, has been created.
- The layout of each page has been implemented to match the provided wireframes.
- Custom styles have been added to enhance the visual appeal of the website.
- React basic styling techniques, such as inline styles, importing CSS files, and utilizing pre-processed CSS files, have been employed.
- For each page, a `<Route>` component has been created to handle the routing.
- Navigation links for all three routes are now displayed consistently across all pages.
- React Router has been utilized to ensure proper functionality and navigation among the different routes.

### CODE REVIEW

File: src/components/Home.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    <p className="home__body-text">
+      Unleash the full potential of your mathematical prowess with our state-of-the-art calculator,
+      it&apos;s your trusted companion for all your numerical adventures. Embrace the thrill of
+      problem-solving and witness the beauty of precision unfold before your eyes.
+    </p>
+    <p className="home__body-text">
+      But Math Magicians offers more than just calculations; it&apos;s a sanctuary for dreamers and
+      visionaries. Delve into our treasury of quotes, where imagination takes center stage. Let the
+      words of great minds transport you to extraordinary realms, igniting the spark of creativity
+      within you.
```
Comment (Munsa1) [inline src/components/Home.js:26]: - [ ] Great work on this project the design is really good, however, we are supposed to follow the wireframe as it guides us on how to lay out our elements on our page. Kindly ensure your design matches the given wireframe in terms of layout.

Your design
image

Given wireframe
image

- Kindly align your heading and paragraph to the left as is in the given design

File: src/components/Calculator.js [hunk 2]
```diff
@@ -71,24 +84,36 @@ export default function Calculator() {
...
+        <div id="calculator__keypad" className="grid">
+          <div id="calculator__grey-btns-container" className="grid">
+            <UtilityPad />
+            <NumericPad />
+          </div>
+          <div id="calculator__orange-btns-container" className="grid">
+            <OperatorsPad />
+          </div>
         </div>
       </div>
```
Comment (Munsa1) [inline src/components/Calculator.js:116]: - [ ] Kindly align the text `Lets do some math` on the calculator page to the left of the page to match the given design

Your design
image

Given frame
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @grauJavier, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Achieved SPA :heavy_check_mark: 
Unique UI Design :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ahmetbozaci** [comment]
### Hello @grauJavier 
### _This is an additional review as a result of the second opinion request._ 

- According to the requirement the general layout should match these [wireframes](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md#:~:text=The%20general%20layout%20should%20match%20these%20wireframes%3A) 
- Your website already matches the wireframe and you added some styles and these are enough for get approved.
 _**Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approved right away**_



___


### Cheers and Happy coding! 💻 👏
____
[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @grauJavier , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- Your design is visually appealing. ✔
- Well done on successfully completing all of the requirements..  🎉 🎉 🎉 ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**